This header file of the tidyhtml library has been renamed in version 5.0.0.
See https://github.com/htacg/tidy-html5/issues/224.
--- a/session/html_secure_tidy.c
+++ b/session/html_secure_tidy.c
@@ -12,7 +12,7 @@
 #include <tidy.h>
 #endif
 
-#include <buffio.h>
+#include <tidybuffio.h>
 
 /* ====================================================================== */
 
